### PR TITLE
V1.3 Red/Green color coded applicant card based on if they have interview scheduled

### LINKED ISSUE #66: As a Hiring Manager receiving many applications, I want to see different color coding of applicants based on if they have an interview scheduled, so that I can tell who has yet to schedule an interview with me

### PR DESCRIPTION
**Pull request by Chen1x**
closes #66

### CODE REVIEW

**Review by Jonyxzx — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/model/interview/Interview.java [hunk 1]
```diff
@@ -40,7 +40,7 @@ public Interview(Applicant app, String role, String timing, boolean isDone) {
 
     /**
      * Returns true if both Interviews have the same Applicant & Timing or if both Interviews are the same object
-     * Adapted from AB3's Person.isSamePerson() method
+     * Adapted from AB3's Applicant.isSameApplicant() method
```
Comment (ghost) [inline src/main/java/seedu/address/model/interview/Interview.java]: I think the comment only makes sense with the original 'AB3's Person.isSamePerson() method, could possibly be reverted?

File: src/main/java/seedu/address/model/interview/UniqueInterviewList.java [hunk 2]
```diff
@@ -19,7 +19,7 @@
  * However, the removal of an interview uses Interview#equals(Object) so as to ensure that the interview
  * with exactly the same fields will be removed.
  * Supports a minimal set of list operations.
- * Adapted from AB3 UniquePersonList
+ * Adapted from AB3 UniqueApplicantList
```
Comment (ghost) [inline src/main/java/seedu/address/model/interview/UniqueInterviewList.java]: Same as the first review comments

File: src/main/java/seedu/address/storage/JsonAdaptedInterview.java [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 /**
  * Jackson-friendly version of {@link seedu.address.model.interview.Interview}.
- * Adapted from AB3's JsonAdaptedPerson
+ * Adapted from AB3's JsonAdaptedApplicant
```
Comment (ghost) [inline src/main/java/seedu/address/storage/JsonAdaptedInterview.java]: Same as the first review comments

File: src/test/java/seedu/address/model/interview/InterviewTest.java [hunk 4]
```diff
@@ -12,7 +12,7 @@
 
 /**
  * Tests for the Interview object methods
- * Adapted from AB3 PersonTest class
+ * Adapted from AB3 ApplicantTest class
```
Comment (ghost) [inline src/test/java/seedu/address/model/interview/InterviewTest.java]: Same as the first review comments

File: src/test/java/seedu/address/testutil/TypicalInterviews.java [hunk 5]
```diff
@@ -8,7 +8,7 @@
 
 /**
  * A utility class containing a list of {@code Interview} objects to be used in tests.
- * Adapted from AB3 TypicalPersons class.
+ * Adapted from AB3 TypicalApplicants class.
```
Comment (ghost) [inline src/test/java/seedu/address/testutil/TypicalInterviews.java]: Same as the first review comments

**Review by ghost — APPROVED** [review]

**Pull request merged**